inngest-core/src/main/kotlin/com/inngest/Function.kt

```diff
@@ -33,6 +33,7 @@ enum class OpCode {
 enum class ResultStatusCode(val code: Int, val message: String) {
     StepComplete(206, "Step Complete"),
     FunctionComplete(200, "Function Complete"),
+    BadRequest(400, "Bad Request"),
```

Comment:
Probably still better to call this `NonRetriableError` just to make it clear as an Enum. `BadRequest` doesn't really say anything.